sormas-ui/src/main/java/de/symeda/sormas/ui/AboutView.java

```diff
@@ -135,22 +135,20 @@ private VerticalLayout createInfoSection() {
 
 		if (InfoProvider.get().isSnapshotVersion()) {
 			Link commitLink = new Link(
-				String.format(
-					"%s (%s)",
-					versionLabel.getValue(),
-					InfoProvider.get()
-						.getLastCommitShortId()),
+				String.format("%s (%s)", versionLabel.getValue(), InfoProvider.get().getLastCommitShortId()),
 				new ExternalResource(InfoProvider.get().getLastCommitHistoryUrl()));
 			commitLink.setTargetName("_blank");
 			CssStyles.style(commitLink, CssStyles.VSPACE_3);
 			infoLayout.replaceComponent(versionLabel, commitLink);
 		}
 
 		if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.LAB_MESSAGES)) {
-			addExternalServiceVersion(
-				Captions.aboutLabMessageAdapter,
-				() -> FacadeProvider.getLabMessageFacade().getLabMessagesAdapterVersion(),
-				infoLayout);
+			if (UserProvider.getCurrent().hasAllUserRights(UserRight.LAB_MESSAGES)) {
```

Comment:
this condition can be added in the outside `if` with `&&` instead of a nested `if`

Reply:
On a second thought it probably makes sense to allow all user rights to view the version of the labmessageadapter. So I suggest removing the user right condition check and add the `@PermitAll` to `LabMessageFacadeEjb::getLabMessagesAdapterVersion`